### PR TITLE
bincount

### PR DESCRIPTION
**Pull request by teckno**

### CODE REVIEW

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by ahmedo42** [comment]
Hi, Can you revert all of the changes unrelated to the `bincount()` function? , the easiest way to do this is to merge master into your branch ( or copy-paste just those 5 files) and commit them

File: ivy/functional/frontends/torch/other_operations .py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+import ivy
+# function for bincount
+def bincount(input, weights=None, minlength=0):
+    return ivy.bincount(input, weights=weights, minlength=minlength)
```
Comment (ahmedo42) [inline ivy/functional/frontends/torch/other_operations .py:4]: the functional API doesn't have a `bincount` function you'll need to implement it compositionally

**Review by ahmedo42 — CHANGES_REQUESTED** [review]
Please remove anything unrelated to the `bincount` function , you can reset the entire branch even and just commit the two files you worked on

File: ivy/functional/frontends/torch/other_operations .py (hunk 1 above)
Comment (teckno) [reply ivy/functional/frontends/torch/other_operations .py:4]: or an example?

**Pull request closed without merging**